src/Fetch_Command.php

```diff
@@ -29,17 +29,23 @@ class Fetch_Command extends WP_CLI_Command {
 	 * [--post-id=<id>]
 	 * : Cache oEmbed response for a given post.
 	 *
+	 * [--discover]
+	 * : Enable oEmbed discovery. Defaults to true.
+	 *
 	 * [--skip-cache]
 	 * : Ignore already cached oEmbed responses. Has no effect if using the 'raw' option, which doesn't use the cache.
 	 *
-	 * [--discover]
-	 * : Enable oEmbed discovery. Defaults to true.
+	 * [--skip-sanitation]
```

Comment:
Please rename this parameter to `--skip-sanitization` (and adapt all references accordingly), otherwise we're in a different business all of a sudden.

See https://wikidiff.com/sanitization/sanitation

Reply:
Lol, what am I like....?!!!